cookbook/navigation/returning-data.md

```diff
@@ -170,13 +170,11 @@ update the `_navigateAndDisplaySelection` method in our `SelectionButton`.
 _navigateAndDisplaySelection(BuildContext context) async {
   final result = await Navigator.push(
     context,
-    new MaterialPageRoute(builder: (context) => new SelectionScreen()),
+    MaterialPageRoute(builder: (context) => SelectionScreen()),
   );
 
   // After the Selection Screen returns a result, show it in a Snackbar!
-  Scaffold
-      .of(context)
-      .showSnackBar(new SnackBar(content: new Text("$result")));
+  Scaffold.of(context).showSnackBar(SnackBar(content: Text("$result")));
```

Comment:
Why is this formatted differently? Did you run it through the formatter?

Reply:
Yep! That was caught by the formatter on TravisCI and needed to change it up so the build would pass.